src/utils/Util.js

```diff
@@ -0,0 +1,42 @@
+import ERROR from '../constants/Error.js';
+import NUMBER from '../constants/Number.js';
+import CHARACTER from '../constants/Character.js';
+
+function throwError(message, condition) {
+  if (condition) {
+    return;
+  }
+  throw new Error(`${ERROR.PREFIX}${message}${ERROR.TRY_AGAIN}`);
+}
+
+const Conditions = {
+  isPresent(value) {
+    return value !== CHARACTER.EMPTY;
+  },
+
+  isMoreThanOne(value) {
+    return value.includes(CHARACTER.COMMA);
+  },
+
+  isContainSpace(value) {
+    return !/\s/.test(value);
+  },
+
+  isCorrectLength(value) {
+    return value.length <= NUMBER.MAX_LENGTH;
+  },
+
+  isDuplicate(arr) {
+    return new Set(arr).size === arr.length;
+  },
+
+  isNumber(value) {
+    return !Number.isNaN(Number(value));
```

Comment:
isNaN 이 이미 내부적으로 Number() 를 수행하기 때문에 다음과 같이 줄여쓸 수 있을 것 같습니닷! ㅎㅎ

```suggestion
    return !Number.isNaN(value);
```